src/Functions/oor.m

```diff
@@ -1,17 +1,30 @@
-%% oor - optimal opliptimal oblique sampling along a given polynonimal
+function [improvedSkeleton, medIntensity,orthIntensVol, orthSamplePointsVol, skelScaleMm] = oor(r3polyToUse, STEP_SIZE, XGrid, YGrid, interpolationF)
+% oor - optimal oblique resampling along a given polynonimal
 %
-% Returns: improvedSkeleton - skeleton gained along the polynomial by
-%                             resampling and intensity weithed centroid calculation
-%          medIntensity     - median intensity profile along the polynoial
+% USAGE:
 %
-% Andreas Husch
-% Centre Hospitalier de Luxembourg / Luxembourg Centre for Systems
-% Biomedicine, University of Luxembourg
-% 2016  - 2017
-% mail@andreashusch.de
+%    [improvedSkeleton, medIntensity, orthIntensVol, orthSamplePointsVol, skelScaleMm] 
+%    = oor(r3polyToUse, STEP_SIZE, XGrid, YGrid, interpolationF)
 %
-% That code needs some cleanup :-) But it works ;-)
-function [improvedSkeleton, medIntensity,orthIntensVol, orthSamplePointsVol, skelScaleMm] = oor(r3polyToUse, STEP_SIZE, XGrid, YGrid, interpolationF)
+% INPUTS: 
+%    r3polyToUse:       
```

Comment:
@adhusch Could you please provide a description for these variables?

Reply:
Inputs:
r3polyToUse: polynomial coefficient matrix
STEP_SIZE: step size along polynomial
Xgrid: grid to sample perpentidular to polyonial (x)
YGrid: grid to sample perpentidular to polyonial (y)
interpolationF: an interpolater fitted to point cloud (see scatteredInterpolant)

Reply:
Outputs: 
improvedSkeleton: newly determined skeleton point cloud after re-sampling
medIntensity: median intensity along the polynomial
orthIntensVol: intensity volume stacked as orthogonal matrix
orthSamplePointsVols: sampling points stacked as orthogonal matrix
skelScaleMm: estimated milimetric scale

Reply:
Thank you!